README.md

```diff
@@ -83,10 +89,12 @@ _As many of these as possible are “statistical” questions – a company may
 * How is the office space physically organized? Is it an open environment? Do developers get offices?
 * What type of hardware is provided? How often is it upgraded?
 * Are developers allowed to use whichever tools they need for the job? Will the company provide funding for these tools?
+* Does IT/Development follow same policies and procedures with regards to hardware choices; operating system; permissions and admin rights on issued hardware; antivirus, DLP, and other security software; office suites; mandated software or "standard" installation?
```

Comment:
Maybe re-word this a little, as I'm not sure what it's asking

Reply:
What I'm getting at is if devs/techies have to follow same rules as e.g. salespeople, analysts, etc. who are less tech saavy. At my current company for instance night support people and IT people have the same procurement standards, firewall/proxy rules, and it's infuriating. Let me think about it and reword it

Reply:
Maybe word it a bit more generically. What are the general harware and software policies? Do I get to choose my own hardware? My own software? Do I have admin rights on my system?

Maybe less wordy than that but you get the idea.